### DISCUSSION TITLE
Small change to enhanced CRM container feature

### DISCUSSION THREAD

**Discussion by iam-steve** [twilio-professional-services/flex-project-template#159] [Q&A] [answered]
Can anyone help guide with a simple modification? Would really love to enable the enhanced CRM container, but we need to be able to load a default URL if there are no tasks selected. Right now, it just shows an empty container where the iframe container would be if there are no tasks reserved or assigned. Any ideas what needs to be update?

Thank you!!

**Comment by jared-hunter** [comment, marked as answer]
hi @iam-steve ive created a PR #214 to illustrate this as well as make it readily available in the future
